### PR TITLE
TweetBot v2

### PR DESCRIPTION
**Pull request by onifs10**

### CODE REVIEW

**Comment by onifs10** [comment]
Screenshot from 2020-07-16 21-45-10

**Comment by goodhands** [comment]
This is a good start, but Functions of Government is a category of budgets and so the `project_name` column should have more description on what the money was used for.

File: app/Http/Controllers/TwitterBot.php [hunk 1]
```diff
@@ -102,4 +105,31 @@ private function filterPaymentTweet($payment)
...
+    public function budgetTweetFilter($budget)
+    {
+        $code = explode('-', $budget->code);
+        if (count($code) > 1) {
+            $ministry_code = $code[1];
+            $sector_code = $code[0];
+            $ministry = Ministry::where('shortname', 'LIKE', "{$ministry_code}%")->first();
+            $sector = Sector::whereId($sector_code)->first();
+            if (is_null($sector)) {
+                return 'the amount of ₦'.$budget->amount." was allocated for ".$budget->classification." in the ".$budget->year ." budget";
```
Comment (goodhands) [inline app/Http/Controllers/TwitterBot.php]: Also try to start the sentences with Uppercase.

**Review by goodhands — CHANGES_REQUESTED** [review]
Good job so far, you rock ✨👓

File: app/Http/Controllers/TwitterBot.php (hunk 1 above)
Comment (onifs10) [reply app/Http/Controllers/TwitterBot.php]: okay

**Comment by onifs10** [comment]
Screenshot from 2020-07-16 22-10-01
check this

**Pull request merged**